tests/uefi_sb/test_uefistored_sb.py

```diff
@@ -210,71 +209,5 @@ def setup_and_cleanup(self, uefi_vm_and_snapshot):
     def test_key_exchanges(self, uefi_vm):
         vm = uefi_vm
 
-        PK = EFIAuth('PK')
-        null_PK = EFIAuth('PK', is_null=True)
-        new_PK = EFIAuth('PK')
-        bad_PK = EFIAuth('PK')
-
-        KEK = EFIAuth('KEK')
-        null_KEK = EFIAuth('KEK', is_null=True)
-
-        db_from_KEK = EFIAuth('db')
-        db_from_PK = EFIAuth('db')
-        null_db_from_KEK = EFIAuth('db', is_null=True)
-        null_db_from_PK = EFIAuth('db', is_null=True)
-
-        PK.sign_auth(PK)
-        PK.sign_auth(null_PK)
-        PK.sign_auth(KEK)
-        PK.sign_auth(null_KEK)
-        PK.sign_auth(new_PK)
-        PK.sign_auth(db_from_PK)
-        PK.sign_auth(null_db_from_PK)
-        PK.sign_auth(db_from_KEK)
-        PK.sign_auth(null_db_from_KEK)
-        KEK.sign_auth(db_from_KEK)
-        KEK.sign_auth(null_db_from_KEK)
-        bad_PK.sign_auth(bad_PK)
-
-        vm.start()
-        vm.wait_for_vm_running_and_ssh_up()
-
-        # at this point we should have a VM with no certs, on a pool with no certs either
-
-        tests = [
-            # Set the PK
-            (PK, True),
-            # Clear the PK
-            (null_PK, True),
-            # Set the PK again
-            (PK, True),
-            # Set a PK with the wrong sig, should fail and PK should be unchanged
-            (bad_PK, False),
-            # Set, clear, and reset the KEK
-            (KEK, True),
-            (null_KEK, True),
-            (KEK, True),
-            # Set and clear the db signed by the KEK
-            (db_from_KEK, True),
-            (null_db_from_KEK, True),
-            # Set and clear the db signed by the PK
-            (db_from_PK, True),
-            (null_db_from_PK, True),
-            # Set a new PK
-            (new_PK, True),
-            # Set old PK, should fail due to expired timestamp
-            (PK, False),
-        ]
-
-        for i, (auth, should_succeed) in enumerate(tests):
-            logging.info('> Testing {} ({})'.format(auth.name, i))
-
-            ok = True
-            try:
-                vm.set_efi_var(auth.name, auth.guid,
-                               EFI_AT_ATTRS_BYTES, auth.auth_data)
-            except SSHCommandFailed:
-                ok = False
-
-            if (should_succeed and not ok) or (ok and not should_succeed):
-                raise AssertionError('Failed to set {} {}'.format(i, auth.name))
+        from .utils import _test_key_exchanges
```

Comment:
It is generally considered bad practice to import anywhere else than at top of file